README.md

```diff
@@ -24,6 +24,10 @@ pip install recommonmark
 Then add this to your Sphinx conf.py:
 
 ```
+# for Sphinx-1.4 or newer
+extensions = ['recommonmark']
```

Comment:
I think this setting is much simple and easy for users. So I hope new package including this change will be released soon.